src/nanodbc.h

```diff
@@ -491,15 +501,18 @@ class statement
     //! \param batch_operations Numbers of rows to fetch per rowset, or the number of batch parameters to process.
     //! \param timeout The number in seconds before query timeout. Default is 0 indicating no timeout.
     //! \return A result set object.
-    //! \attention You will want to use transactions if you are doing batch operations because it will prevent auto commits from occurring after each individual operation is executed.
+    //! \attention You will want to use transactions if you are doing batch operations because it will prevent auto
+    //!            commits from occurring after each individual operation is executed.
```

Comment:
@lexicalunit check that multi line comments like this are processed correctly by doxygen.